### PR TITLE
- adds language related to the apiDeploymentBaseUrl property

### PR DESCRIPTION
**Pull request by baywet**
related https://github.com/microsoft/OpenApi.ApiManifest/pull/7
@peombwa I didn't follow the second subpoint of the second open question in your implementation PR. Would you mind expanding please?

### CODE REVIEW

**Review by peombwa — APPROVED** [review]

File: draft-miller-api-manifest.md [hunk 1]
```diff
@@ -79,15 +79,15 @@ The publisher object MUST contain a `name` property that is a JSON string. This
...
+Each API dependency object represents an HTTP API that the target application consumes. The API dependency object MAY contain a `apiDescriptionUrl` that references an API description document such as an [OpenAPI](https://spec.openapis.org/oas/latest.html) description. The `apiDeploymentBaseUrl` member MAY contain the base URL to use in combination with request info{{requestInfo}} `uriTemplate` properties. When the base url is used, it MUST match one of the servers entries in the OpenAPI description.  The `apiDescriptionVersion` member can contain the version of the API Description used by the application. This member enables tooling to detect if the referenced API description is updated. The `authorizationRequirements` property contains the requirements for the target application to authorize a call to the HTTP API. The `requests` property contains an array of `requestInfo` objects.
```
Comment (darrelmiller) [inline draft-miller-api-manifest.md:82]: We need to be really, really careful in our wording around how this baseUrl is combined with path in the request Info object.  I don't want to bring forward the mistake made in OpenAPI where relative paths used for pathItem keys start with /.

Comment (baywet) [reply draft-miller-api-manifest.md:82]: Do you want me to include language saying it MUST end with a trailing slash?

Comment (peombwa) [reply draft-miller-api-manifest.md:82]: > Do you want me to include language saying it MUST end with a trailing slash?

This sounds about right to me. I'll let Darrel have the final say on the wording. Trailing slash validation will also need to be added to the library if we settle for a trailing slash in `apiDeploymentBaseUrl`.

Comment (darrelmiller) [reply draft-miller-api-manifest.md:82]: Yup. I'm onboard with that strategy.

Comment (baywet) [reply draft-miller-api-manifest.md]: added, let me know what you think.

**Pull request merged**